BlazarService/src/main/java/com/hubspot/blazar/resources/ModuleBuildResource.java

```diff
@@ -155,6 +168,42 @@ public long getSize() {
     };
   }
 
+  @GET
+  @Path("{id}/log/download")
```

Comment:
@Path("{id}/log/download-url")
public String getBuildLogDownloadUrl
It would be fine to send a redirect once in a while but an endpoint that sends only redirects doesn't seem very clean to me. I think that it is cleaner for the endpoint to be assumed that it generates the log download url. The client can then decide what to do with it. Also the combination of get and /download is ugly and confusing.